alfredo-zimperz/precourse/html-calculator/html-calculator.js

```diff
@@ -0,0 +1,217 @@
+// ----- *** config & initial settings *** ------
+
+const MAX_DECIMAL_NUMBERS = 2
+let auxNumber = ''
+let auxDecimalNumber = ''
+let firstNumber = null
+let secondNumber = null
+let hasDecimal = false
+let operator = null
+let lastResult = null
+let lastResultIsPressed = false
+let operatorIsPressed = false
+
+// ----- *** getting elements *** ------
+
```

Comment:
lo que comentamos ayer de los espacios tras for, function, if, else, etc.... y recuerda primero constantes y despues variables

Reply:
corregido